service/history/transferQueueActiveProcessor.go

```diff
@@ -87,7 +87,7 @@ func newTransferQueueActiveProcessor(
 		MetricScope:                        metrics.TransferActiveQueueProcessorScope,
 	}
 	currentClusterName := shard.GetService().GetClusterMetadata().GetCurrentClusterName()
-	logger = logger.WithTags(tag.ClusterName(currentClusterName))
+	logger = logger.WithTags(tag.ClusterName(currentClusterName), tag.ComponentTransferQueue)
```

Comment:
I think this is not needed, because it is already tagged at `newTransferQueueProcessor`

Reply:
Good catch. Removed the tag